.github/workflows/release-dry-run.yml

```
@@ -0,0 +1,139 @@
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+          
+      - uses: jdx/mise-action@v2
+          
+      - name: Check version
+        id: check
+        run: |
+          # Run mise task and capture output
+          output=$(mise run release:check-version)
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)